CONTRIBUTING.md

```diff
@@ -163,6 +170,20 @@ For votes regarding non-coding issues such as procedure changes, the requirement
 that a veto is accompanied by a *technical* justification is relaxed somewhat,
 though a well reasoned justification must still be included.
 
+### Reversions
+
+A *-1* vote on any recently merged pull request is grounds for an immediate
```

Comment:
@erasche Can you also fix https://github.com/galaxyproject/galaxy/pull/314#discussion_r31823679 ?